.github/ISSUE_TEMPLATE/funding_form.yml

```
@@ -13,6 +13,8 @@ body:
         Fill in this form to get funding approved for your event.
 
         Make sure to read <https://nixos.org/community/event-funding.html> before filling the form.
+
+        We ask to publish a picture of the group on social media to spread the word about Nix.
```

I think overall this is fine, but just making sure it isn't a requirement, as some people don't want to get photographed and posted online.

---

In that case the person can move aside so others can take a picture.

It's not strictly mandatory, but we would like to see that a) there's some proof the event happened and b) there's a word out these events are happening :)

---

The goal is to gather attention. It could also be in the form of a well-written report. But a picture is usually easier.

---

How about

```suggestion
        We ask to publish a witness of the event on social media (be it a group photo, a written report, slides, or anything) to spread the word about Nix.
```

to make it more flexible while keeping the intent?